client/state/domains/management/reducer.js

```diff
@@ -72,22 +78,18 @@ export const saving = createReducer( false, {
  * @return {Object}        Updated state
  */
 export const items = createReducer( {}, {
-	[ DOMAIN_MANAGEMENT_CONTACT_DETAILS_CACHE_RECEIVE ]: ( state, { cacheData } ) => ( { ...state, _contactDetailsCache: cacheData } ),
-	[ DOMAIN_MANAGEMENT_CONTACT_DETAILS_CACHE_UPDATE ]:
-		( state, { cacheData } ) =>
-			merge( {}, state, { _contactDetailsCache: cacheData } ),
+	[ DOMAIN_MANAGEMENT_CONTACT_DETAILS_CACHE_RECEIVE ]: ( state, { data } ) => ( { ...state, _contactDetailsCache: data } ),
+	[ DOMAIN_MANAGEMENT_CONTACT_DETAILS_CACHE_UPDATE ]: ( state, { data } ) => {
+		return merge( {}, state, { _contactDetailsCache: data } );
```

Comment:
`merge` is kind of the bulldozer of object transforms :), mainly because it's a recursive operation. I don't think it's appropriate in this case, as the following would be equivalent:

```js
( state, { data } ) => ( { ...state, _contactDetailsCache: data } )
```